### DISCUSSION TITLE
Defensive drones cant navigate upwards

### DISCUSSION THREAD

**Discussion by FredrikM97** [FakeFishGames/Barotrauma#14448] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Defensive drones cant navigate upwards and cant open doors. If I have a door for a lower level and it is open the drones will fall down/navigate down. However, they cant follow upwards and usually get stuck in the ballast. This is with the R29 sub. They cant navigate up in the large ballast either and get stuck in the lower floor where they have no usage.

### Reproduction steps

Create drones, open a door going down, make them follow you and then try to climb the ladder again.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Just fixed actually https://github.com/FakeFishGames/Barotrauma/issues/14321 https://github.com/FakeFishGames/Barotrauma/issues/14322
Mb will be included in autumn update

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/14321 and https://github.com/FakeFishGames/Barotrauma/issues/14322. This has been just addressed in the unstable build, and the changes will be coming up in the next update. :)
